firefox-ios/Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -60,6 +60,12 @@ class TabScrollingController: NSObject, FeatureFlaggable, SearchBarLocationProvi
         return isBottomSearchBar ? bottomShowing : headerTopOffset == 0
     }
 
+    private var shouldSetInitialScrollToTop: Bool {
+        return tab?.mimeType == MIMEType.PDF && shouldScrollToTop
+    }
```

Comment:
Seems like this method makes it so there's two sources of truth for whether we should scroll to the top. Could we just conditionally modify `shouldScrollToTop` when it's a PDF instead of this?

Reply:
@dataports May you do here, a code suggestion, please? Thank you.

Reply:
good question 😆 I tried to find a way to do what I was thinking of and there wasn't really a good one. This should be fine.